packages/typespec-ts/test/modularIntegration/authOauth2.spec.ts

```diff
@@ -57,7 +57,8 @@ describe("OAuth2Context in API Layer", () => {
       await invalid(context);
       assert.fail("Expected an exception to be thrown.");
     } catch (err: any) {
-      assert.strictEqual(err.error, "invalid-grant");
+      console.log(err);
+      assert.strictEqual(err.message, "Unexpected status code: 403");
```

Comment:
could we add some assertion for the whole erro object?

Reply:
iirc, a few years ago, there's some security concern that we should not include raw response in the error as of privacy related concerns ?

Reply:
iirc, a few years ago, there's some security concern that we should not include raw response in the error as of privacy related concerns ?

Reply:
I think if we use `@azure/logger` and a level of `verbose` we can sidestep some of those concerns since it wouldn't show up in logs unless explicitly enabled